index.html

```diff
@@ -823,7 +823,7 @@
       <li id="step-response-end">Record the time as <a for="PerformanceResourceTiming">responseEnd</a>
       immediately after receiving the last byte of the response.
       <ol>
-        <li>Return to step <a href="#step-connect-start">13</a> if the user agent fails to send the request or receive the entire response, and needs to reopen the connection.
+        <li>Return to step <a href="#step-connect-start">12</a> if the user agent fails to send the request or receive the entire response, and needs to reopen the connection.

```

Comment:
What if the UA does not try to reopen the connection... e.g. closes it or times out on the response? Should we say something like "...if the user agent does not attempt to reopen the connection proceed to step 17?"